in_toto/verifylib.py

```diff
@@ -37,10 +37,49 @@
 import in_toto.runlib
 import in_toto.models.layout
 import in_toto.models.link
-from in_toto.exceptions import (RuleVerficationError, LayoutExpiredError)
+from in_toto.exceptions import (RuleVerficationError, LayoutExpiredError,
+    BadReturnValueError)
 from in_toto.matchrule_validators import check_matchrule_syntax
 import in_toto.log as log
 
+def _raise_on_bad_retval(return_value, command=None):
```

Comment:
According to our code-style-guidelines its not required. But I guess it does not hurt either.